src/auth/axios.ts

```diff
@@ -19,6 +18,7 @@ const listener = () => {
 };
 
 store.subscribe(listener);
```

Comment:
Is this used anywhere? I got an error when it wasn't commented out

Reply:
Hmm, it is, I'll look into this right now it came up yesterday for me too